src/core/database/query/matching.ts

```diff
@@ -54,7 +57,7 @@ export const matchProps = (options: MatchPropsOptions = {}) => {
           [
             node(nodeName),
             relation('out', 'r', { active: !changeset }),
-            node('prop', 'Property'),
+            node('prop', deleted ? 'Deleted_Property' : 'Property'),
```

Comment:
I don't think this is necessary. When we delete nodes we leave the properties alone (active). It's only when properties change that we do this.

Reply:
If the node is deleted, its properties are moved to `Deleted_Property` and cannot find with the `Property` label. Can you check on your side?

Reply:
Ah you are right. Labels change. The relation stays active true though, that's what was thinking of.